doc/Overview/Mvux/Advanced/Pagination.md

```diff
@@ -0,0 +1,398 @@
+
+### View
+
+```xml
+<Page 
+    x:Class="PaginationPeopleApp.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:PaginationPeopleApp"
+    xmlns:mvux="using:Uno.Extensions.Reactive.UI">
+
+    <Page.Resources>
+        <DataTemplate x:Key="PersonDataTemplate" x:DataType="local:Person">
+            <StackPanel Orientation="Horizontal" Spacing="2">
+                <TextBlock Text="{x:Bind FirstName}"/>
+                <TextBlock Text="{x:Bind LastName}"/>
+            </StackPanel>
+        </DataTemplate>
+    </Page.Resources>
+
+    <ListView ItemsSource="{Binding PeopleAuto}" ItemTemplate="{StaticResource PersonDataTemplate}"
+        DataFetchSize="20" Header="Next page auto-loading when on available capacity or scrolled"/>
+
+</Page>
+```
+
+In addition, make sure the `DataContext` of the `Page` is set to the generated bindable model called `BindablePeopleModel` (*MainPage.xaml.cs*):
+
+```csharp
+public MainPage()
+{
+    this.InitializeComponent();
+
+    this.DataContext = new BindablePeopleModel(new PeopleService());
+}
+```
+
+> [!TIP]  
+> You can inspect the generated code by either placing the cursor on the word `BindablePeopleModel` and hitting <kbd>F12</kbd>, see other ways to inspect the generated code [here](xref:Overview.Mvux.Advanced.InspectGeneratedCode).
+
+As you can see, there's nothing special in the XAML code as MVUX is taking advantage of the tools already implemented with the `ListView`.
+
+- When the page load, the first 20 items are loaded (after a delay of one second - as simulated in the service).
+- If there's available space in the `ListView`, the next batch of 20 items will be requested from the Feed and loaded from the service thereafter.
+- When the user scroll down and hits the bottom of the `ListView`, the next 20 items will be requested.
+- This behavior will follow as the user scrolls down and chases the service for more data until all items have been loaded.
+
+Here's what the app renders like:
+
+![](../Assets/PaginationIncrementalLoading.gif)
+
+> [!TIP]  
+> You can use these members of the `ListView` to control data loading: [`DataFetchSize`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.listviewbase.datafetchsize), [`IncrementalLoadingThreshold`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.listviewbase.incrementalloadingthreshold), and [`IncrementalLoadingTrigger`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.listviewbase.incrementalloadingthreshold).
```

Comment:
Not sure all of this is supported by Uno, did you validated? I think that we should not promote them if they are not.